src/services/tx/tx-sender/dispatch.ts

```diff
@@ -85,10 +86,12 @@ export const dispatchTxSigning = async (
   try {
     signedTx = await tryOffChainTxSigning(safeTx, safeVersion, sdk)
   } catch (error) {
-    txDispatch(TxEvent.SIGN_FAILED, {
-      txId,
-      error: asError(error),
-    })
+    if ((error as EthersError).code !== ErrorCode.ACTION_REJECTED) {
```

Comment:
This covers signatures but there's also executions and relays and other types of actions (basically all dispatch functions in this file).